### ISSUE TITLE
Implement localised fires from International Handbook of Structural Fire Engineering

### ISSUE THREAD

**Issue by simonsantama** [fire-library/openfire#14]
Document: International Handbook of Structural Fire Engineering.
Reference: Chapter 4. Section 4.5
Equations 4.11 and 4.12 (which require an implementation of 4.13 and 4.14). Also equations 4.21 and 4.22 (which require 4.15 to 4.19)

**Comment by JamieMaclean** [comment]
@simonsantama I have lots of questions on this one. We will need to spend some time together on this one I think.

**Comment by simonsantama** [comment]
Yes. Can you also have a read at the section after on travelling fires?
That one will also require is to go through it together I think
